src/main/java/io/kestra/plugin/influxdb/FluxQuery.java

```diff
@@ -0,0 +1,295 @@
+                        |> filter(fn: (r) => r._measurement == "cpu")
+                        |> yield()
+                    fetchType: STORE
+                """
+        ),
+        @Example(
+            title = "Query data from InfluxDB and return all rows in execution output",
+            full = true,
+            code = """
+                id: influxdb_flux_query_inline
+                namespace: company.team
+
+                tasks:
+                  - id: query_influxdb
+                    type: io.kestra.plugin.influxdb.FluxQuery
+                    connection:
+                      url: "{{ secret('INFLUXDB_URL') }}"
+                      token: "{{ secret('INFLUXDB_TOKEN') }}"
+                    org: "my-org"
+                    query: |
+                      from(bucket: "my-bucket")
+                        |> range(start: -1h)
+                        |> filter(fn: (r) => r._measurement == "cpu")
+                        |> limit(n: 10)
+                        |> yield()
+                    fetchType: FETCH
+                """
+        ),
+        @Example(
+            title = "Query data from InfluxDB and return only the first row",
+            full = true,
+            code = """
+                id: influxdb_flux_query_one
+                namespace: company.team
+
+                tasks:
+                  - id: query_influxdb
+                    type: io.kestra.plugin.influxdb.FluxQuery
+                    connection:
+                      url: "{{ secret('INFLUXDB_URL') }}"
+                      token: "{{ secret('INFLUXDB_TOKEN') }}"
+                    org: "my-org"
+                    query: |
+                      from(bucket: "my-bucket")
+                        |> range(start: -1h)
+                        |> filter(fn: (r) => r._measurement == "cpu")
+                        |> limit(n: 1)
+                        |> yield()
+                    fetchType: FETCH_ONE
+                """
+        )
+    }
+)
+public class FluxQuery extends AbstractTask implements RunnableTask<FluxQuery.Output> {
+    @Schema(
+        title = "Flux query to execute",
+        description = "The Flux query to run against InfluxDB"
+    )
+    @NotNull
+    @PluginProperty(dynamic = true)
+    private Property<String> query;
+
+    @Schema(
+        title = "Whether to store the data from the query result into an ion serialized data file."
+    )
+    @Builder.Default
+    private Property<Boolean> store = Property.of(false);
+
+    @Schema(
+        title = "The way you want to store the data.",
+        description = "FETCH_ONE output the first row, "
+            + "FETCH output all the rows, "
+            + "STORE store all rows in a file, "
+            + "NONE do nothing."
+    )
+    @PluginProperty
+    @Builder.Default
+    private Property<FetchType> fetchType = Property.of(FetchType.FETCH);
+
+    public enum FetchType {
+        FETCH,
+        FETCH_ONE,
+        STORE,
+        NONE
+    }
```

Comment:
Please use `io.kestra.core.models.tasks.common.FetchType` instead of declaring your own.